features/step_definitions/additional_cli_steps.rb

```diff
@@ -10,7 +10,7 @@
     if RUBY_VERSION == '1.8.7' && string =~ /\{.+=>.+\}/
       warn "Skipping checking #{string} on 1.8.7 because hash ordering is not consistent"
     else
-      assert_partial_output(string, all_output)
+      expect(last_command_stopped).to have_output Regexp.new(Regexp.escape(extract_text(unescape_text(string))))
```

Comment:
So has `assert_partial_output` been removed?  This new form seems much much messier, given that it requires:

* `unescape_text`
* `extract_text`
* `Regexp.escape`
* `Regexp.new`

...all passed to `have_output`.

I don't think I'd ever be able to come up with that and even now I don't understand the reason for all of it.